templates/html-css-js-static/README.md

```diff
@@ -0,0 +1,50 @@
+# TITLE
+
+<!--
+
+Describe the exercise in a few sentences. E.g.:
+
+This challenge offers a simple form with three input fields. Let's make it interactive with JS!
+
+-->
+
+## Task
+
+<!--
+
+Explaining the task in detail. E.g.:
+
+Use JavaScript to react to the form submission.
+
+Log all form data (in object form) into the console in the submit event handler.
+
+You can use the following hints as guideline:
+
+- Hint 1
+- Hint 2
+- ...
+
+Switch to the [index.js](./js/index.js) file and make something great happen!
+
+-->
+
+## Notes
+
+- You only have to touch the `js/index.js` file.
```

Comment:
(This applies to all template readme files)

I think we should be consistent on how we build a reference to another file from the readme. Right now we have:

Task section
- display filename
- linked with relative path
- not formatted as inline code

Notes section
- display filename with relative path
- no link
- formatted as inline code


Currently it's also inconsistent in the existing exercises

Reply:
👍  lets discuss this @ 10